src/components/form/new.vue

```diff
@@ -224,6 +224,9 @@ export default {
               actual: details.value
             });
           }
+          if (code === 409.17) {
+            return `${this.$tcn('util.request.problem.409_17', details.duplicateProperties.length)}\n\n${details.duplicateProperties.map(p => `• ${p.provided} (${this.$t('common.existing')}: ${p.current})`).join('\n')}`;
```

Comment:
It looks like this is the same in `FormNew` and `FormPublish`. If a 409.17 will always be handled in the same way, I think we can go ahead and move that logic to a centralized place in `requestAlertMessage()`. We do something similar for 404.1 Problems currently.

Reply:
In addition to `FormNew` and `FormPublish`, isn't this Problem a possibility for dataset/overview/new-property.vue? That could be another reason to centralize it.